pkg/storaged/overview.jsx

```diff
@@ -49,7 +49,7 @@ export class Overview extends React.Component {
                         <Gallery hasGutter>
                             <Card>
                                 <CardBody>
-                                    <StoragePlots client={client} onHover={(dev) => this.setState({ highlight: dev })} />
+                                    <StoragePlots client={client} className="storage-graphs" onHover={(dev) => this.setState({ highlight: dev })} />
```

Comment:
This just adds a class which is not corresponding to some rule.
Maybe you forgot something?